src/async/executor.rs

```diff
@@ -36,7 +37,9 @@ impl Alarm {
             let ptr = Box::into_raw(tag);
             let timeout = GprTimespec::inf_future();
             let cq_ref = cq.borrow()?;
-            grpc_sys::grpc_alarm_create(cq_ref.as_ptr(), timeout, ptr as _)
```

Comment:
But why it compiles?

Reply:
Why can't it compile?

Reply:
The signature has changed.

Reply:
The signature of C/C++ code changes, not the API written in rust. Rust compiler can just compile rust code.

Reply:
So why does the rustc find the changed C API?

Reply:
Because they share the same name. See also #130.